README.md

```diff
@@ -7,7 +7,7 @@ Deploy a FastHTML application to [HuggingFace Spaces](https://huggingface.co/spa
 1. Create a free account on [HuggingFace](https://huggingface.co)
 2. Go to your account settings and create an access token with write access. Keep this token safe and don't share it.
 3. Set the `HF_TOKEN` environment variable to that token
-4. Install fasthtml-hf: `pip install fasthtml-hf`
+4. Add `fasthtml-hf` to your requirements.txt and install it: `pip install fasthtml-hf`.
```

Comment:
I wasn't 100% sure of the intent behind the `requirements.txt` file in the repo so that's why I left it unchanged and simply added this line.

Reply:
I don't think you can remove the pip install line though - you need both. But yes actually changing requirements.txt here probably better

Reply:
True. I re-added the original line and created a separate line about adding `fasthtml-hf` to requirements.txt.